collections/concepts/what-are-fingerprints.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "Fingerprints"
+categories: concepts
+permalink: /concepts/what-are-fingerprints/
+nav_order: 3
+has_children: true
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# What Is Fingerprinting?
+
+Tokenizing data replaces raw values with a Token Identifier referencing sensitive data, meaning you don’t have the raw value in your database. Without the raw value, your system isn’t able to perform simple queries that your application may require - For example, what if you need to check for duplicate or existing values? Fingerprints solve this problem by providing you an identifier representing your Token’s uniqueness without needing to read the decrypted data.
+
+<span class="base-alert info">
+  <span>
+    Need access to the data? Our [Reactors](https://developers.basistheory.com/concepts/what-are-reactors/) and [Proxy](https://developers.basistheory.com/concepts/what-is-the-proxy/) enable you to share, compute, and analyze anything with your sensitive data without it touching your systems.
+  </span>
+</span>
+
+## How It Works
+
+Every [Token Type](https://docs.basistheory.com/#token-types) created with the same data will have the same fingerprint. Fingerprints are:
+ - A non-reversible random unique identifier. It is not derived from the data.
+ - Unique to the underlying data 
+ - Unique to the Token Type
+ - Unique to the Tenant
+
+For example, a social security number in a single tenant would generate the same fingerprint every time it is tokenized. For the same value in a different tenant, it would generate a different fingerprint, unique to that tenant.
+
+[Learn which token types support fingerprinting.](https://docs.basistheory.com/#token-types)
+
+Due to its non-sensitive nature, fingerprints can be stored in any environment. Even if the fingerprint were leaked or compromised there is no way to reverse it. Fingerprints are immune to brute force attacks because they are unique to your tenant.
+Take a look at this sample request and response:
+
+```js
+curl "<https://api.basistheory.com/atomic/cards>" \\
+    -H "Content-Type: application/json" \\
+    -H "BT-API-KEY: <BT_API_KEY>" \\
+    -X "POST"  \\
+    -D '{
+		"card": {
+          "number": "4242424242424242",
+          "expiration_month": 12,
+          "expiration_year": 2022,
+          "cvc": "123"
+        }
+	}'
```

Comment:
Is the JSON indentation wrong here or is that just an issue in this view?

Reply:
fixed